.nvmrc

```diff
@@ -1 +1 @@
-v10.16.0
+v20.11.0
```

Comment:
**issue (blocking):** we also should change the node version in the Dockerfile: https://github.com/AlbertHernandez/branch-name-action/blob/106d820c9f9275a7d0f8cf3d4ef637a46bb7b4ba/Dockerfile#L1

Reply:
Changed